stdlib/public/core/IntegerParsing.swift

```diff
@@ -83,17 +83,19 @@ where CodeUnits.Element : UnsignedInteger {
   }
 }
 
-@inline(never)
-internal func _parseASCIIOutlined<
-  CodeUnits : IteratorProtocol, Result: FixedWidthInteger
->(
-  codeUnits: inout CodeUnits, radix: Result
-) -> Result?
-where CodeUnits.Element : UnsignedInteger {
-  return _parseASCII(codeUnits: &codeUnits, radix: radix)
-}
-
 extension FixedWidthInteger {
+  /// _parseASCII function thunk that prevents inlining used as an
```

Comment:
Better, thanks.

Other nits, should you care to address them:

1. This is not a doc comment; it's an implementation comment (a doc comment would begin with a sentence fragment describing what it _does_). Therefore it should not use triple slashes
2. I would prefer if you used something that described the role of this method, such as `SlowPath`, over something like `Outlined`, whose most common meaning is not at all what it's trying to communicate.